### PR TITLE
adaptation for stream input and JSON multiple-datum input

### PR DESCRIPTION
**Pull request by jywarren**
Each newline is sent as a separate request.

### CODE REVIEW

**Comment by rjcorwin** [comment]
Hi @jywarren - I just added you as a collaborator on this project so you can accept your pull requests in case I'm MIA on a river in Oregon again :-).

File: push [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 
+process.stdin.on('error', function(e) {
+  console.log("error: "+e)
+})
 
 if (options.hasOwnProperty('help')) {
   help()
 }
+else if (options.hasOwnProperty('stream')) {
+  process.stdin.pipe(split())
```
Comment (rjcorwin) [inline push:16]: oooh, that looks like a nice trick.

**Comment by jywarren** [comment]
Rad!

**Comment by rjcorwin** [comment]
Do these examples I just added to the README look right to you?

https://github.com/openpipekit/opk-phant-cli/blob/master/README.md

**Comment by jywarren** [comment]
yep!

**Pull request merged**